sztp-agent/pkg/secureAgent/agent.go

```diff
@@ -7,6 +7,27 @@ Copyright (C) 2022 Red Hat.
 
 package secureAgent
 
+const (
+	CONTENT_TYPE_YANG = "Content-Type:application/yang-data+json"
```

Comment:
should be `"application/yang-data+json"` not `"Content-Type:application/yang-data+json"`